bnn_optimization/data/preprocessing/__init__.py

```diff
@@ -0,0 +1,3 @@
+from bnn_optimization.data.preprocessing import efficientnet, vgg
+
+__all__ = ["efficientnet", "vgg"]
```

Comment:
We don't use `vgg` so we should remove it. Also should we rename the efficientnet preprocessing to something else since it is basically renset preprocessing and it might be confusing since the efficientnet paper uses autoaugment by default and we don't